js_modules/dagit/packages/core/src/graphql/graphql.ts

```diff
@@ -1490,6 +1497,13 @@ export enum InstigationType {
 
 export type InstigationTypeSpecificData = ScheduleData | SensorData;
 
+export type InstigatorSelector = {
+  instigatorType: Scalars['String'];
+  repositoryLocationName: Scalars['String'];
+  repositoryName: Scalars['String'];
+  scheduleName: Scalars['String'];
```

Comment:
This could be a `sensor` right? Maybe just `name`?

Reply:
yea I think I need to re-run the command - shouldn't still be scheduleName